src/axios/Api.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
-const API = axios.create({
+const Api = axios.create({
```

Comment:
This should be lowercase as uppercase is reserved for components, also this file should be lowercase and .js rather than jsx